microsoft-edge/extensions-chromium/developer-guide/sidebar.md

```diff
@@ -5,7 +5,7 @@ author: MSEdgeTeam
 ms.author: msedgedevrel
 ms.topic: conceptual
 ms.prod: microsoft-edge
-ms.date: 06/27/2023
+ms.date: 07/01/2023
 ---
 # Extensions in the Microsoft Edge sidebar
 
```

Comment:
Is it ok to delete this sentence? --
This is a forthcoming feature; sidebar extensions are not yet in a stable release of Microsoft Edge.

@abhishek-shanthkumar, @Bhuvana-Priya

Rendered article section: first sentence of article:
https://review.learn.microsoft.com/en-us/microsoft-edge/extensions-chromium/developer-guide/sidebar?branch=pr-en-us-2677

Reply:
replaced by fresh [Comment](https://github.com/MicrosoftDocs/edge-developer/pull/2677/files#r1294071890)